doc/manuals/orion-api.md

```diff
@@ -2156,81 +2199,88 @@ The values that `options` parameter can have for this specific request are:
+    "dataProvided": {
+      "entities": [
+        {
+          "id": "Bcn_Welt",
+          "type": "Room"
+        }
+      ],
+      "attrs": [
+        "temperature"
+      ]
+    },
+    "provider": {
+      "http": {
+        "url": "http://contextsource.example.org"
+      },
+      "supportedForwardingMode": "all"
+    },
+    "expires": "2017-10-31T12:00:00",
+    "status": "active",
+    "forwardingInformation": {
+      "timesSent": 12,
+      "lastForwarding": "2017-10-06T16:00:00.00Z",
+      "lastSuccess": "2017-10-06T16:00:00.00Z",
+      "lastFailure": "2017-10-05T16:00:00.00Z"
+    }
+  }
+]
+```
 
-        [
-          {
-            "id": "62aa3d3ac734067e6f0d0871",
-            "description": "Example Context Source",
-            "dataProvided": {
-              "entities": [
-                {
-                  "id": "Bcn_Welt",
-                  "type": "Room"
-                }
-              ],
-              "attrs": [
-                "temperature"
-              ]
-            },
-            "provider": {
-              "http": {
-                "url": "http://contextsource.example.org"
-              },
-              "supportedForwardingMode": "all"
-            },
-            "expires": "2017-10-31T12:00:00",
-            "status": "active",
-            "forwardingInformation": {
-              "timesSent": 12,
-              "lastForwarding": "2017-10-06T16:00:00.00Z",
-              "lastSuccess": "2017-10-06T16:00:00.00Z",
-              "lastFailure": "2017-10-05T16:00:00.00Z"
-            }
-          }
-        ]
 
 #### Create Registration [POST /v2/registrations]
 
 Creates a new context provider registration. This is typically used for binding context sources
 as providers of certain data.
 The registration is represented by a JSON object as described at the beginning of this section.
 
-Response:
+**Request payload** 
+
+Content-Type is `application/json``
+
+```json
+{
+  "description": "Relative Humidity Context Source",
+  "dataProvided": {
+    "entities": [
+      {
+        "id": "room2",
+        "type": "Room"
+      }
+    ],
+    "attrs": [
+      "relativeHumidity"
+    ]
+  },
+  "provider": {
+    "http":{ 
+      "url": "http://localhost:1234"
+    }
+  }
+}
+```
+
+**Response**
 
 * Successful operation uses 201 Created
 * Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
   more details.
 
-+ Request (application/json)
-
-        {
-          "description": "Relative Humidity Context Source",
-          "dataProvided": {
-            "entities": [
-              {
-                "id": "room2",
-                "type": "Room"
-              }
-            ],
-            "attrs": [
-              "relativeHumidity"
-            ]
-          },
-          "provider": {
-            "http":{ 
-              "url": "http://localhost:1234"
-            }
-          }
-        }
-
-+ Response 201
+Example response 201:
 
     + Headers
 
```

Comment:
This part renders in a weird way

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/40356341/176891122-5d64c707-da75-431c-a480-e402a62e8926.png">